tsconfigdoc.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://json.schemastore.org/tsconfig",
   "extends": "./tsconfigbase",
+  "exclude": ["**/test/**", "**/testutils/**"],
```

Comment:
I think we can also delete the exclude settings in typedoc.js.

Reply:
I agree. But I still want to know more about exactly what `exclude` in `tsconfigdoc.json` (which presumably is just acting as the standard tsconfig option here) is supposed to do vs `exclude` in typedoc-specific config